<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [DOCKTESTERS] BWA-Mem validation of DO51057 normal) 0.013% miss-matches, and 3.7% soft-matches, tumor) 0.043% miss-matches, and 4.64% soft-matches
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:docktesters%40lists.icgc.org?Subject=Re%3A%20%5BDOCKTESTERS%5D%20BWA-Mem%20validation%20of%20DO51057%20normal%29%200.013%25%0A%20miss-matches%2C%20and%203.7%25%20soft-matches%2C%20tumor%29%200.043%25%20miss-matches%2C%0A%20and%204.64%25%20soft-matches&In-Reply-To=%3CCA%2Bdx8mazZRmaEJt_E5BpPLDmpeCxy5efJMbiv4ez2dPNJOWibQ%40mail.gmail.com%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="000297.html">
   <LINK REL="Next"  HREF="000304.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[DOCKTESTERS] BWA-Mem validation of DO51057 normal) 0.013% miss-matches, and 3.7% soft-matches, tumor) 0.043% miss-matches, and 4.64% soft-matches</H1>
    <B>Miguel Vazquez</B> 
    <A HREF="mailto:docktesters%40lists.icgc.org?Subject=Re%3A%20%5BDOCKTESTERS%5D%20BWA-Mem%20validation%20of%20DO51057%20normal%29%200.013%25%0A%20miss-matches%2C%20and%203.7%25%20soft-matches%2C%20tumor%29%200.043%25%20miss-matches%2C%0A%20and%204.64%25%20soft-matches&In-Reply-To=%3CCA%2Bdx8mazZRmaEJt_E5BpPLDmpeCxy5efJMbiv4ez2dPNJOWibQ%40mail.gmail.com%3E"
       TITLE="[DOCKTESTERS] BWA-Mem validation of DO51057 normal) 0.013% miss-matches, and 3.7% soft-matches, tumor) 0.043% miss-matches, and 4.64% soft-matches">miguel.vazquez at cnio.es
       </A><BR>
    <I>Mon Apr 10 08:15:00 EDT 2017</I>
    <P><UL>
        <LI>Previous message: <A HREF="000297.html">[DOCKTESTERS] BWA-Mem validation of DO51057 (normal BAM only). 96.3% matches, 0.013% miss-matches, and 3.7% soft-matches
</A></li>
        <LI>Next message: <A HREF="000304.html">[DOCKTESTERS] BWA-Mem validation of DO51057 normal) 0.013% miss-matches, and 3.7% soft-matches, tumor) 0.043% miss-matches, and 4.64% soft-matches
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#298">[ date ]</a>
              <a href="thread.html#298">[ thread ]</a>
              <a href="subject.html#298">[ subject ]</a>
              <a href="author.html#298">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>Hi all,

The comparison with the *tumor BAM* for DO51057 has completed with *rates
of miss-maches (**0.043%) and soft-matches (**4.64%) just slightly
higher* *than
for the* *normal BAM*. These *numbers are not definitive* since as you can
read from Jonas just below, *there might still be a discrepancy* in the
order in which the BAM where processed.  We'll soon know from Jonas if a
different order will fix these rates even more.

*Lines*: 1010685786
*Matches*: 963319037
*Misses*: 442926
*Soft*: 46923823

Best regards

Miguel

On Mon, Apr 10, 2017 at 1:14 PM, Jonas Demeulemeester &lt;
<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">Jonas.Demeulemeester at crick.ac.uk</A>&gt; wrote:

&gt;<i> Hi all,
</I>&gt;<i>
</I>&gt;<i> I&#8217;m currently running the comparison of the BWA-Mem docker reproduced bams
</I>&gt;<i> and the PCAWG ones for DO51057.
</I>&gt;<i> I should be able to send a report some time today.
</I>&gt;<i>
</I>&gt;<i> Miguel, looking at your code, I believe you&#8217;re feeding the unaligned bams
</I>&gt;<i> into the pipeline in the order given by the read group lines (@RG) in the
</I>&gt;<i> header of the PCAWG bam.
</I>&gt;<i> I&#8217;m using the order recorded in the command line/programs used (@CL/@PG)
</I>&gt;<i> lines of the PCAWG bam, which is often different for whatever reason.
</I>&gt;<i> I&#8217;m not entirely sure which one is the correct one, but I&#8217;m guessing the
</I>&gt;<i> one in the @CL/@PG lines is the actual one as it chronologically reiterates
</I>&gt;<i> the whole procedure ( [align - sort] x N followed by merge + flag dups )
</I>&gt;<i> If this is the case, the true % mismatches may be lower still than 0.013%,
</I>&gt;<i> if not, then I should see a higher mismatch rate and the 0.013% is due to
</I>&gt;<i> something else still.
</I>&gt;<i>
</I>&gt;<i> Regarding the soft-matches, I agree with Junjun, we may want to ask the
</I>&gt;<i> people behind the variant callers, but I guess they are probably dealing
</I>&gt;<i> with these multiply-mapping reads internally.
</I>&gt;<i>
</I>&gt;<i> Best,
</I>&gt;<i> Jonas
</I>&gt;<i>
</I>&gt;<i>
</I>&gt;<i> _________________________________
</I>&gt;<i> Jonas Demeulemeester, PhD
</I>&gt;<i> Postdoctoral Researcher
</I>&gt;<i> The Francis Crick Institute
</I>&gt;<i> 1 Midland Road
</I>&gt;<i> London
</I>&gt;<i> NW1 1AT
</I>&gt;<i>
</I>&gt;<i> *T:* +44 (0)20 3796 2594 &lt;+44%2020%203796%202594&gt;
</I>&gt;<i> M: +44 (0)7482 070730 &lt;+44%207482%20070730&gt;
</I>&gt;<i> *E:* <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">jonas.demeulemeester at crick.ac.uk</A>
</I>&gt;<i> *W:* www.crick.ac.uk
</I>&gt;<i>
</I>&gt;<i>
</I>&gt;<i>
</I>&gt;<i> On 9 Apr 2017, at 15:47, Junjun Zhang &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">Junjun.Zhang at oicr.on.ca</A>&gt; wrote:
</I>&gt;<i>
</I>&gt;<i> Hi Miguel,
</I>&gt;<i>
</I>&gt;<i> This is indeed good news, the mismatch is significantly lower.
</I>&gt;<i>
</I>&gt;<i> Regarding soft matches, thanks for the explanation. I wonder whether it
</I>&gt;<i> has impact (or how much impact) on variant calls, do variant callers take
</I>&gt;<i> into account the information that a read may map to multiple places? Does
</I>&gt;<i> it make adjustment at the time of variant calling? I guess these are
</I>&gt;<i> questions for variant caller authors.
</I>&gt;<i>
</I>&gt;<i> Thanks,
</I>&gt;<i> Junjun
</I>&gt;<i>
</I>&gt;<i>
</I>&gt;<i>
</I>&gt;<i> From: &lt;docktesters-bounces+junjun.zhang=<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">oicr.on.ca at lists.icgc.org</A>&gt; on
</I>&gt;<i> behalf of Miguel Vazquez &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">miguel.vazquez at cnio.es</A>&gt;
</I>&gt;<i> Date: Thursday, April 6, 2017 at 11:36 AM
</I>&gt;<i> To: Lincoln Stein &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">lincoln.stein at gmail.com</A>&gt;
</I>&gt;<i> Cc: Francis Ouellette &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">francis at oicr.on.ca</A>&gt;, Keiran Raine &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A>&gt;,
</I>&gt;<i> &quot;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A>&quot; &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A>&gt;
</I>&gt;<i> Subject: Re: [DOCKTESTERS] BWA-Mem validation of DO51057 (normal BAM
</I>&gt;<i> only). 96.3% matches, 0.013% miss-matches, and 3.7% soft-matches
</I>&gt;<i>
</I>&gt;<i> Hi Lincoln,
</I>&gt;<i>
</I>&gt;<i> Soft-match means that the alignment position in the new BAM is not the
</I>&gt;<i> same is the one in the original BAM, but is included in the list of
</I>&gt;<i> alternative alignments for that read.
</I>&gt;<i>
</I>&gt;<i> For instance, the original bam aligns a read to chr 1 pos 1000, but also
</I>&gt;<i> admits that is could be aligned at chr 2 pos 2000 or chr 3 pos 3000, the
</I>&gt;<i> new bam aligns it at chr 2 pos 2000, which is not the position chosen by
</I>&gt;<i> the original BAM but is in the alternative list. It could also work the
</I>&gt;<i> other way, that the original position is included in the list of
</I>&gt;<i> alternative positions of the new BAM
</I>&gt;<i>
</I>&gt;<i> I hope this was clear.
</I>&gt;<i>
</I>&gt;<i> Best regards
</I>&gt;<i>
</I>&gt;<i> Miguel
</I>&gt;<i>
</I>&gt;<i>
</I>&gt;<i>
</I>&gt;<i>
</I>&gt;<i>
</I>&gt;<i>
</I>&gt;<i> On Thu, Apr 6, 2017 at 4:55 PM, Lincoln Stein &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">lincoln.stein at gmail.com</A>&gt;
</I>&gt;<i> wrote:
</I>&gt;<i>
</I>&gt;&gt;<i> Hi Miguel,
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Sounds like a significant achievement! But remind me what a &quot;soft match&quot;
</I>&gt;&gt;<i> is?
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Lincoln
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> On Thu, Apr 6, 2017 at 10:28 AM, Miguel Vazquez &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">miguel.vazquez at cnio.es</A>&gt;
</I>&gt;&gt;<i> wrote:
</I>&gt;&gt;<i>
</I>&gt;&gt;&gt;<i> Dear all,
</I>&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;<i> This is just an advance teaser for the BWA-Mem validation after the
</I>&gt;&gt;&gt;<i> latest changes, it is currently running over the tumor BAM, but the normal
</I>&gt;&gt;&gt;<i> BAM has completed and the *missmatches are two orders of magnitude
</I>&gt;&gt;&gt;<i> lower* than in our two previous attempts. Before further discussion
</I>&gt;&gt;&gt;<i> here are the raw numbers:
</I>&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;<i> Lines: 1125172217
</I>&gt;&gt;&gt;<i> Matches: 1083221794
</I>&gt;&gt;&gt;<i> *Misses: 143716*
</I>&gt;&gt;&gt;<i> Soft: 41806707
</I>&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;<i> If my calculation are correct this means 96.3% matches, *0.013%
</I>&gt;&gt;&gt;<i> miss-matches*, and 3.7% soft-matches.
</I>&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;<i> The fix was two part. First realizing that the input of this process
</I>&gt;&gt;&gt;<i> should not be a single unaligned version of the output BAMs, but several
</I>&gt;&gt;&gt;<i> input BAMs. Breaking down the output bam into it's constituent BAMs, by a
</I>&gt;&gt;&gt;<i> process implemented by Jonas, dit not address the problem unfortunately.
</I>&gt;&gt;&gt;<i> After this first attempt it was pointed out to us, I think by Keiran, that
</I>&gt;&gt;&gt;<i> the order of the reads matter, and so our attempt to work back from the
</I>&gt;&gt;&gt;<i> output BAM was not going to work. Junjun came back to us with the second
</I>&gt;&gt;&gt;<i> part of the fix, he located a subset of original unaligned BAMs in the DKFZ
</I>&gt;&gt;&gt;<i> that we could use. Downloading these BAM files and submitting them to
</I>&gt;&gt;&gt;<i> BWA-Mem in the same order as was specified in the output BAM header
</I>&gt;&gt;&gt;<i> achieved these promising results.
</I>&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;<i> I will reply this message in a few days with the corresponding numbers
</I>&gt;&gt;&gt;<i> for the other BAM, the tumor, which is currently running.
</I>&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;<i> Best regards
</I>&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;<i> Miguel
</I>&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;<i> On Sun, Feb 19, 2017 at 1:43 PM, Miguel Vazquez &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">miguel.vazquez at cnio.es</A>&gt;
</I>&gt;&gt;&gt;<i> wrote:
</I>&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;<i> Dear all,
</I>&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;<i> Great news! The BWA-Mem test on a real PCAWG donor succeed in running;
</I>&gt;&gt;&gt;&gt;<i> achieving an overlap with the original BAM alignment similar to the
</I>&gt;&gt;&gt;&gt;<i> HCC1143 test. The numbers are:
</I>&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;<i> Lines: 1708047647
</I>&gt;&gt;&gt;&gt;<i> Matches: 1589172843
</I>&gt;&gt;&gt;&gt;<i> Misses: 62726130
</I>&gt;&gt;&gt;&gt;<i> Soft: 56148674
</I>&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;<i> Which mean 93% matches, 3.6% miss-matches, and 3.2% soft-matches.
</I>&gt;&gt;&gt;&gt;<i> Compared to the HCC1143 test there are a few percentage points in matches
</I>&gt;&gt;&gt;&gt;<i> that turn into soft-matches (95% and 1.3% to 93% and 3.2%), but the ratio
</I>&gt;&gt;&gt;&gt;<i> of misses is very close 3.6%.
</I>&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;<i> I'm running this test on a second donor.
</I>&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;<i> Best regards
</I>&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;<i> Miguel
</I>&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;<i> On Tue, Feb 14, 2017 at 3:30 PM, Miguel Vazquez &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">miguel.vazquez at cnio.es</A>
</I>&gt;&gt;&gt;&gt;<i> &gt; wrote:
</I>&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;<i> Dear colleagues,
</I>&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;<i> I'm very happy to say that the BWA-Mem pipeline finished for the
</I>&gt;&gt;&gt;&gt;&gt;<i> HCC1143 data.
</I>&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;<i> I think what solved the problem was setting the headers to the
</I>&gt;&gt;&gt;&gt;&gt;<i> unaligned BAM files. I'm currently trying it out with the DO35937 donor,
</I>&gt;&gt;&gt;&gt;&gt;<i> but its too early to say if its working or not.
</I>&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;<i> To compare BAM files I've followed some advice that I found on the
</I>&gt;&gt;&gt;&gt;&gt;<i> internet <A HREF="https://www.biostars.org/p/166221/.">https://www.biostars.org/p/166221/.</A> I will detail them a bit
</I>&gt;&gt;&gt;&gt;&gt;<i> below because I would like some advice as to how appropriate the approach
</I>&gt;&gt;&gt;&gt;&gt;<i> is, but first here are the numbers:
</I>&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;<i> *Lines*: 74264390
</I>&gt;&gt;&gt;&gt;&gt;<i> *Matches*: 70565742
</I>&gt;&gt;&gt;&gt;&gt;<i> *Misses*: 2693687
</I>&gt;&gt;&gt;&gt;&gt;<i> *Soft*: 1004961
</I>&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;<i> Which means *95% matches, 3.6% miss-matches, and 1.3% soft-matches*.
</I>&gt;&gt;&gt;&gt;&gt;<i> Matches are when the chromosome and position are the same, soft-matches are
</I>&gt;&gt;&gt;&gt;&gt;<i> when they are not the same but the position from one of the alignments is
</I>&gt;&gt;&gt;&gt;&gt;<i> included in the list of alternative positions for the other alignment (e.g
</I>&gt;&gt;&gt;&gt;&gt;<i> XA:Z:15,-102516528,76M,0), and misses are the rest.
</I>&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;<i> Here is the detailed process from the start. The comparison script is
</I>&gt;&gt;&gt;&gt;&gt;<i> here <A HREF="https://github.com/mikisvaz/PCAWG-Docker-Test/blob/master/bi">https://github.com/mikisvaz/PCAWG-Docker-Test/blob/master/bi</A>
</I>&gt;&gt;&gt;&gt;&gt;<i> n/compare_bwa_bam.sh
</I>&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;<i> 1) Un-align tumor and normal BAM files, retaining the original aligned
</I>&gt;&gt;&gt;&gt;&gt;<i> BAM files
</I>&gt;&gt;&gt;&gt;&gt;<i> 2) Run BWA-Mem wich produces a file called HCC1143.merged_output.bam
</I>&gt;&gt;&gt;&gt;&gt;<i> with alignments from both tumor and normal
</I>&gt;&gt;&gt;&gt;&gt;<i> 3) use samtools to extract the entries, limited for the first in pair
</I>&gt;&gt;&gt;&gt;&gt;<i> (?), cut the read-name, chromosome, position (??) and extra information
</I>&gt;&gt;&gt;&gt;&gt;<i> (for additional alignments) and sort them. We do this for the original
</I>&gt;&gt;&gt;&gt;&gt;<i> files and for the BWA-Mem merged_output file, but separating tumor and
</I>&gt;&gt;&gt;&gt;&gt;<i> normal entries (marked with the codes 'tumor' and 'normal', I believe from
</I>&gt;&gt;&gt;&gt;&gt;<i> the headers I set when un-aligning them)
</I>&gt;&gt;&gt;&gt;&gt;<i> 4) join the lines by read-name, separately for the tumor and normal
</I>&gt;&gt;&gt;&gt;&gt;<i> pairs of files, and check for matches
</I>&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;<i> I've two questions:
</I>&gt;&gt;&gt;&gt;&gt;<i> (?) Is it OK to select only the first in pair, its what the guy in the
</I>&gt;&gt;&gt;&gt;&gt;<i> example did, and it did simplify the code without repeated read-names
</I>&gt;&gt;&gt;&gt;&gt;<i> (??) I guess its OK to only check chromosome and position, the cigar
</I>&gt;&gt;&gt;&gt;&gt;<i> would be necessarily the same.
</I>&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;<i> Best regards
</I>&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;<i> Miguel
</I>&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;<i> On Mon, Jan 16, 2017 at 3:24 PM, Miguel Vazquez &lt;
</I>&gt;&gt;&gt;&gt;&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">miguel.vazquez at cnio.es</A>&gt; wrote:
</I>&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Dear all,
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Let me summarize the status of the testing for Sanger and DKFZ. The
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> validation has been run for two donors for each workflow: DO50311 DO52140
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Sanger:
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ----------
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Sanger call only somatic variants. The results are *identical for
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Indels and SVs* but *almost identical for SNV.MNV and CNV*. The
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> discrepancies are reproducible (on the same machine at least), i.e. the
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> same are found after running the workflow a second time.
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> DKFZ:
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---------
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> DKFZ cals somatic and germline variants, except germline CNVs. For
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> both germline and somatic variants the results are *identical for
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> SNV.MNV and Indels* but with *large discrepancies for SV and CNV*.
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Kortine Kleinheinz and Joachim Weischenfeldt are in the process of
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> investigating this issue I believe.
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> BWA-Mem failed for me and has also failed for Denis Yuen and Jonas
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Demeulemeester. Denis I believe is investigating this problem further. I
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> haven't had the chance to investigate this much myself.
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Best
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Miguel
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---------------------
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> RESULTS
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---------------------
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">ubuntu at ip-10-253-35-14</A>:~/DockerTest-Miguel$ cat results.txt
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.snv.mnv for DO50311 using DKFZ
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 51087
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.indel for DO50311 using DKFZ
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 26469
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.sv for DO50311 using DKFZ
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 231
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 44
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 10:20596800:N:&lt;TRA&gt;,10:5606682
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> 1:N:&lt;TRA&gt;,11:16776092:N:&lt;TRA&gt;
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 48
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 10:119704959:N:&lt;INV&gt;,10:131163
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> 22:N:&lt;TRA&gt;,10:47063485:N:&lt;TRA&gt;
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.cnv for DO50311 using DKFZ
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 731
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 213
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 10:132510034:N:&lt;DEL&gt;,10:205968
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> 01:N:&lt;NEUTRAL&gt;,10:47674883:N:&lt;NEUTRAL&gt;
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 190
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 10:100891940:N:&lt;NEUTRAL&gt;,10:10
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> 4975905:N:&lt;NEUTRAL&gt;,10:119704960:N:&lt;NEUTRAL&gt;
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of germline.snv.mnv for DO50311 using DKFZ
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 3850992
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of germline.indel for DO50311 using DKFZ
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 709060
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of germline.sv for DO50311 using DKFZ
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 1393
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 231
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 10:134319313:N:&lt;DEL&gt;,10:134948
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> 976:N:&lt;DEL&gt;,10:19996638:N:&lt;DEL&gt;
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 615
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 10:101851839:N:&lt;TRA&gt;,10:101851
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> 884:N:&lt;TRA&gt;,10:10745225:N:&lt;DUP&gt;
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> File not found /mnt/1TB/work/DockerTest-Migue
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> l/tests/DKFZ/DO50311//output//DO50311.germline.cnv.vcf.gz
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.snv.mnv for DO52140 using DKFZ
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 37160
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.indel for DO52140 using DKFZ
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 19347
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.sv for DO52140 using DKFZ
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 72
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 23
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 10:132840774:N:&lt;DEL&gt;,11:382520
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> 19:N:&lt;TRA&gt;,11:47700673:N:&lt;TRA&gt;
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 61
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 10:134749140:N:&lt;DEL&gt;,11:179191
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> :N:&lt;TRA&gt;,11:38252005:N:&lt;TRA&gt;
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.cnv for DO52140 using DKFZ
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 275
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 94
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 1:106505931:N:&lt;LOH&gt;,1:10906889
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> 9:N:&lt;DEL&gt;,1:109359995:N:&lt;DEL&gt;
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 286
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 10:88653561:N:&lt;LOH&gt;,11:179192:
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> N:&lt;LOH&gt;,11:38252006:N:&lt;LOH&gt;
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of germline.snv.mnv for DO52140 using DKFZ
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 3833896
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of germline.indel for DO52140 using DKFZ
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 706572
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of germline.sv for DO52140 using DKFZ
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 1108
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 1116
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 10:102158308:N:&lt;DEL&gt;,10:104645
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> 247:N:&lt;DEL&gt;,10:105097522:N:&lt;DEL&gt;
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 2908
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 10:100107032:N:&lt;TRA&gt;,10:100107
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> 151:N:&lt;TRA&gt;,10:102158345:N:&lt;DEL&gt;
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> File not found /mnt/1TB/work/DockerTest-Migue
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> l/tests/DKFZ/DO52140//output//DO52140.germline.cnv.vcf.gz
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.snv.mnv for DO50311 using Sanger
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 156299
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 1
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: Y:58885197:A:G
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 14
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 1:102887902:A:T,1:143165228:C:G,16:87047601:A:C
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.indel for DO50311 using Sanger
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 812487
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.sv for DO50311 using Sanger
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 260
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.cnv for DO50311 using Sanger
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 138
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.snv.mnv for DO52140 using Sanger
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 87234
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 5
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 1:23719098:A:G,12:43715930:T:A,20:4058335:T:A
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 7
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 10:6881937:A:T,1:148579866:A:G,11:9271589:T:A
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.indel for DO52140 using Sanger
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 803986
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.sv for DO52140 using Sanger
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 6
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Comparison of somatic.cnv for DO52140 using Sanger
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> ---
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Common: 36
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Extra: 0
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i> Missing: 2
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>     - Example: 10:11767915:T:&lt;CNV&gt;,10:11779907:G:&lt;CNV&gt;
</I>&gt;&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;&gt;<i>
</I>&gt;&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> --
</I>&gt;&gt;<i> *Lincoln Stein*
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Scientific Director (Interim), Ontario Institute for Cancer Research
</I>&gt;&gt;<i> Director, Informatics and Bio-computing Program, OICR
</I>&gt;&gt;<i> Senior Principal Investigator, OICR
</I>&gt;&gt;<i> Professor, Department of Molecular Genetics, University of Toronto
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> &lt;<A HREF="http://goog_1828306398/">http://goog_1828306398/</A>&gt;
</I>&gt;&gt;<i> *Ontario Institute for Cancer Research*
</I>&gt;&gt;<i> MaRS Centre
</I>&gt;&gt;<i> 661 University Avenue
</I>&gt;&gt;<i> Suite 510
</I>&gt;&gt;<i> Toronto, Ontario
</I>&gt;&gt;<i> Canada M5G 0A3
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Tel: 416-673-8514
</I>&gt;&gt;<i> Mobile: 416-817-8240
</I>&gt;&gt;<i> Email: <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">lincoln.stein at gmail.com</A>
</I>&gt;&gt;<i> Toll-free: 1-866-678-6427
</I>&gt;&gt;<i> Twitter: @OICR_news
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> *Executive Assistant*
</I>&gt;&gt;<i> *Melisa Torres*
</I>&gt;&gt;<i> Tel:  647-259-4253
</I>&gt;&gt;<i> Email: <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">melisa.torres at oicr.on.ca</A> &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">stacey.quinn at oicr.on.ca</A>&gt;
</I>&gt;&gt;<i> www.oicr.on.ca
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> This message and any attachments may contain confidential and/or
</I>&gt;&gt;<i> privileged information for the sole use of the intended recipient. Any
</I>&gt;&gt;<i> review or distribution by anyone other than the person for whom it was
</I>&gt;&gt;<i> originally intended is strictly prohibited. If you have received this
</I>&gt;&gt;<i> message in error, please contact the sender and delete all copies.
</I>&gt;&gt;<i> Opinions, conclusions or other information contained in this message may
</I>&gt;&gt;<i> not be that of the organization.
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> _______________________________________________
</I>&gt;&gt;<i> docktesters mailing list
</I>&gt;&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A>
</I>&gt;&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">https://lists.icgc.org/mailman/listinfo/docktesters</A>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;<i> _______________________________________________
</I>&gt;<i> docktesters mailing list
</I>&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A>
</I>&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">https://lists.icgc.org/mailman/listinfo/docktesters</A>
</I>&gt;<i>
</I>&gt;<i>
</I>&gt;<i> The Francis Crick Institute Limited is a registered charity in England and
</I>&gt;<i> Wales no. 1140062 and a company registered in England and Wales no.
</I>&gt;<i> 06885462, with its registered office at 1 Midland Road London NW1 1AT
</I>&gt;<i>
</I>&gt;<i> _______________________________________________
</I>&gt;<i> docktesters mailing list
</I>&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A>
</I>&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">https://lists.icgc.org/mailman/listinfo/docktesters</A>
</I>&gt;<i>
</I>&gt;<i>
</I>-------------- next part --------------
An HTML attachment was scrubbed...
URL: &lt;<A HREF="https://lists.icgc.org/mailman/private/docktesters/attachments/20170410/4cc83fb3/attachment-0001.html">https://lists.icgc.org/mailman/private/docktesters/attachments/20170410/4cc83fb3/attachment-0001.html</A>&gt;
</PRE>



<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="000297.html">[DOCKTESTERS] BWA-Mem validation of DO51057 (normal BAM only). 96.3% matches, 0.013% miss-matches, and 3.7% soft-matches
</A></li>
	<LI>Next message: <A HREF="000304.html">[DOCKTESTERS] BWA-Mem validation of DO51057 normal) 0.013% miss-matches, and 3.7% soft-matches, tumor) 0.043% miss-matches, and 4.64% soft-matches
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#298">[ date ]</a>
              <a href="thread.html#298">[ thread ]</a>
              <a href="subject.html#298">[ subject ]</a>
              <a href="author.html#298">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="https://lists.icgc.org/mailman/listinfo/docktesters">More information about the docktesters
mailing list</a><br>
</body></html>
