<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [DOCKTESTERS] Sanger symlinked input file issue
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:docktesters%40lists.icgc.org?Subject=Re%3A%20%5BDOCKTESTERS%5D%20Sanger%20symlinked%20input%20file%20issue&In-Reply-To=%3CDF1B9B0F-1659-4E7C-B049-56E52EE387CE%40sanger.ac.uk%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="000130.html">
   <LINK REL="Next"  HREF="000133.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[DOCKTESTERS] Sanger symlinked input file issue</H1>
    <B>Keiran Raine</B> 
    <A HREF="mailto:docktesters%40lists.icgc.org?Subject=Re%3A%20%5BDOCKTESTERS%5D%20Sanger%20symlinked%20input%20file%20issue&In-Reply-To=%3CDF1B9B0F-1659-4E7C-B049-56E52EE387CE%40sanger.ac.uk%3E"
       TITLE="[DOCKTESTERS] Sanger symlinked input file issue">kr2 at sanger.ac.uk
       </A><BR>
    <I>Thu Oct 20 04:28:12 EDT 2016</I>
    <P><UL>
        <LI>Previous message: <A HREF="000130.html">[DOCKTESTERS] Sanger symlinked input file issue
</A></li>
        <LI>Next message: <A HREF="000133.html">[DOCKTESTERS] Broad PCAWG Tokens definition?
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#131">[ date ]</a>
              <a href="thread.html#131">[ thread ]</a>
              <a href="subject.html#131">[ subject ]</a>
              <a href="author.html#131">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>Hi Denis,

Less painful than I thought.

Just update line 163 of the dockerfile to pull this version:

<A HREF="https://github.com/ICGC-TCGA-PanCancer/CGP-Somatic-Docker/blob/develop/Dockerfile">https://github.com/ICGC-TCGA-PanCancer/CGP-Somatic-Docker/blob/develop/Dockerfile</A> &lt;<A HREF="https://github.com/ICGC-TCGA-PanCancer/CGP-Somatic-Docker/blob/develop/Dockerfile">https://github.com/ICGC-TCGA-PanCancer/CGP-Somatic-Docker/blob/develop/Dockerfile</A>&gt;

Regards,

Keiran Raine
Principal Bioinformatician
Cancer Genome Project
Wellcome Trust Sanger Institute

<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A>
Tel:+44 (0)1223 834244 Ext: 4983
Office: H104

&gt;<i> On 19 Oct 2016, at 21:36, Denis Yuen &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">Denis.Yuen at oicr.on.ca</A>&gt; wrote:
</I>&gt;<i> 
</I>&gt;<i> Hi,
</I>&gt;<i> 
</I>&gt;<i> FYI, breaking this out into a thread so hopefully this renders better in threaded email clients. 
</I>&gt;<i> 
</I>&gt;<i> Keiran, I took a look at running that /usr/bin/perl /opt/wtsi-cgp/bin/pindel_input_gen.pl command while pointing it at the symlinked version in /var/spool/cwl and surprisingly it works (or at least executes well past the displayed error). 
</I>&gt;<i> 
</I>&gt;<i> Unfortunately, this command is generated and it doesn't seem to be part of the CGP-Somatic-Docker &lt;<A HREF="https://github.com/ICGC-TCGA-PanCancer/CGP-Somatic-Dockerhttp://">https://github.com/ICGC-TCGA-PanCancer/CGP-Somatic-Dockerhttp://</A>&gt; codebase. 
</I>&gt;<i> Basically, the SeqWare code that is in the that repo encodes this command (note that it refers to the version in /var/spool/cwl)
</I>&gt;<i> 
</I>&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">seqware at 83bd876f1b5d</A>:/datastore/oozie-76d2e11f-9e92-44aa-841e-a7ddf70a4aab/generated-scripts$ cat s58_cgpPindel_input_70.sh
</I>&gt;<i> #!/usr/bin/env bash
</I>&gt;<i> set -o errexit
</I>&gt;<i> set -o pipefail
</I>&gt;<i> 
</I>&gt;<i> export SEQWARE_SETTINGS=/var/spool/cwl/.seqware/settings
</I>&gt;<i> cd /datastore/oozie-76d2e11f-9e92-44aa-841e-a7ddf70a4aab
</I>&gt;<i> /usr/bin/time /usr/bin/time --format=&quot;Wall_s %e\nUser_s %U\nSystem_s %S\nMax_kb %M&quot; --output=/var/spool/cwl/timings/0_cgpPindel_input_2 /home/seqware/CGP-Somatic-Docker/target/Workflow_Bundle_CgpSomaticCore_0.0.0_SeqWare_1.1.1/Workflow_Bundle_CgpSomaticCore/0.0.0/bin/wrapper.sh /opt/wtsi-cgp pindel.pl -p input -r /var/spool/cwl/reference_files/genome.fa -e MT,GL%,hs37d5,NC_007605 -st WGS -as GRCh37 -sp human -s /var/spool/cwl/reference_files/pindel/simpleRepeats.bed.gz -f /var/spool/cwl/reference_files/pindel/genomicRules.lst -g /var/spool/cwl/reference_files/pindel/human.GRCh37.indelCoding.bed.gz -u /var/spool/cwl/reference_files/pindel/pindel_np.gff3.gz -sf /var/spool/cwl/reference_files/pindel/softRules.lst -b /var/spool/cwl/reference_files/brass/ucscHiDepth_0.01_mrg1000_no_exon_coreChrs.bed.gz -o /var/spool/cwl/0/pindel -t /var/spool/cwl/7875b5196f6b8b52847f99bf370aada0.bam -n /var/spool/cwl/fdcb1bd7cffca69d15383ca9566c58e0.bam -i 2 -c 4
</I>&gt;<i> 
</I>&gt;<i> I think /opt/wtsi-cgp pindel.pl must then be generating the failing command (using  /opt/wtsi-cgp/bin/pindel_input_gen.pl  ) that I listed below with the original symlinked location. 
</I>&gt;<i> I believe that the culprit is this section of pindel.pl which must be tracing the input back to the original target of the symlink and using that instead. 
</I>&gt;<i> 
</I>&gt;<i>   # make all things that appear to be paths absolute
</I>&gt;<i>   for (keys %opts) {
</I>&gt;<i>     $opts{$_} = abs_path($opts{$_}) if(defined $opts{$_} &amp;&amp; -e $opts{$_});
</I>&gt;<i>   }
</I>&gt;<i> 
</I>&gt;<i> Not entirely sure why this affects this particular donor, but that's what I've found so far. Should we attempt to modify pindel or is there a configuration setting we can take advantage of? 
</I>&gt;<i> 
</I>&gt;<i> 
</I>&gt;<i> 
</I>&gt;<i> From: Keiran Raine [<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A> &lt;mailto:<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A>&gt;]
</I>&gt;<i> Sent: October 17, 2016 4:25 AM
</I>&gt;<i> To: Denis Yuen
</I>&gt;<i> Cc: Miguel Vazquez; Adam Struck; <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A> &lt;mailto:<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A>&gt;
</I>&gt;<i> Subject: Re: [DOCKTESTERS] [PAWG-TECH] Draft agenda for PCAWG-TECH teleconference
</I>&gt;<i> 
</I>&gt;<i> Hi Denis,
</I>&gt;<i> 
</I>&gt;<i> The commands executed by the Pindel step appear to be being handed the original BAM file location rather (/var/lib/cwl) than the symlinks in the output area (/var/spool/cwl).  This would be the problem.
</I>&gt;<i> 
</I>&gt;<i> All the jobs that execute after the BAS generation should use the symlinked BAMs in the output area (although I think it's only important for pindel and brass).
</I>&gt;<i> 
</I>&gt;<i> Regards,
</I>&gt;<i> 
</I>&gt;<i> Keiran Raine
</I>&gt;<i> Principal Bioinformatician
</I>&gt;<i> Cancer Genome Project
</I>&gt;<i> Wellcome Trust Sanger Institute
</I>&gt;<i> 
</I>&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A> &lt;x-<A HREF="msg://7/redir.aspx?REF=4zfyWFME5zuNg3Iu2duWBTIs04TZffesqfdqwRC7VeM4Kn9QXvjTCAFtYWlsdG86a3IyQHNhbmdlci5hYy51aw..">msg://7/redir.aspx?REF=4zfyWFME5zuNg3Iu2duWBTIs04TZffesqfdqwRC7VeM4Kn9QXvjTCAFtYWlsdG86a3IyQHNhbmdlci5hYy51aw..</A>&gt;
</I>&gt;<i> Tel:+44 (0)1223 834244 Ext: 4983
</I>&gt;<i> Office: H104
</I>&gt;<i> 
</I>&gt;&gt;<i> On 15 Oct 2016, at 22:10, Denis Yuen &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">Denis.Yuen at oicr.on.ca</A> &lt;x-<A HREF="msg://7/redir.aspx?REF=EEPuTl7_Q12Px7_fSgo92oc2O0HCcrAbJ6FakT0TcB04Kn9QXvjTCAFtYWlsdG86RGVuaXMuWXVlbkBvaWNyLm9uLmNh">msg://7/redir.aspx?REF=EEPuTl7_Q12Px7_fSgo92oc2O0HCcrAbJ6FakT0TcB04Kn9QXvjTCAFtYWlsdG86RGVuaXMuWXVlbkBvaWNyLm9uLmNh</A>&gt;&gt; wrote:
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Hi,
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Agreed, to sum up:
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> 1) The donor test set includes bas files. The real donor sets do not. 
</I>&gt;&gt;<i> 2) That said, the way the CWL file is written, regardless of whether a bas file is provided in the test set, they don't actually make it into the docker container. Instead, they get generated inside the container while it is running.
</I>&gt;&gt;<i> 3) The pindel step does indeed fail in DO50311 on a host that successfully ran the test data. 
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Keiran, some additional info for debugging.
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> The CWL file results in this docker invocation:
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> [job temp8674499429956656923.cwl] /tmp/tmp0NOg7v$ docker \
</I>&gt;&gt;<i>     run \
</I>&gt;&gt;<i>     -i \
</I>&gt;&gt;<i>     --volume=/home/ubuntu/CGP-Somatic-Docker-original/./datastore/launcher-ba5b9c2f-0b64-473b-8c9f-99e1153ad0de/inputs/70fb518e-42c8-4fdd-b473-d3b380aafbdb/fdcb1bd7cffca69d15383ca9566c58e0.bam.bai:/var/lib/cwl/stge94eb32c-dc0f-4523-a88c-d1e00a322537/fdcb1bd7cffca69d15383ca9566c58e0.bam.bai:ro \
</I>&gt;&gt;<i>     --volume=/home/ubuntu/CGP-Somatic-Docker-original/./datastore/launcher-ba5b9c2f-0b64-473b-8c9f-99e1153ad0de/inputs/8bcaff90-24fb-4d61-8898-800a95dce3e0/7875b5196f6b8b52847f99bf370aada0.bam:/var/lib/cwl/stg2e1e9218-4ca7-4cbb-b474-dfdf6ac0b8a6/7875b5196f6b8b52847f99bf370aada0.bam:ro \
</I>&gt;&gt;<i>     --volume=/home/ubuntu/CGP-Somatic-Docker-original/./datastore/launcher-ba5b9c2f-0b64-473b-8c9f-99e1153ad0de/inputs/8bcaff90-24fb-4d61-8898-800a95dce3e0/7875b5196f6b8b52847f99bf370aada0.bam.bai:/var/lib/cwl/stg2e1e9218-4ca7-4cbb-b474-dfdf6ac0b8a6/7875b5196f6b8b52847f99bf370aada0.bam.bai:ro \
</I>&gt;&gt;<i>     --volume=/home/ubuntu/CGP-Somatic-Docker-original/./datastore/launcher-ba5b9c2f-0b64-473b-8c9f-99e1153ad0de/inputs/ab975a04-937f-40fc-b3e5-40b41c2295fc/GRCh37d5_CGP_refBundle.tar.gz:/var/lib/cwl/stg218c30fe-3a28-4c3a-9803-253754dae462/GRCh37d5_CGP_refBundle.tar.gz:ro \
</I>&gt;&gt;<i>     --volume=/home/ubuntu/CGP-Somatic-Docker-original/./datastore/launcher-ba5b9c2f-0b64-473b-8c9f-99e1153ad0de/inputs/70fb518e-42c8-4fdd-b473-d3b380aafbdb/fdcb1bd7cffca69d15383ca9566c58e0.bam:/var/lib/cwl/stge94eb32c-dc0f-4523-a88c-d1e00a322537/fdcb1bd7cffca69d15383ca9566c58e0.bam:ro \
</I>&gt;&gt;<i>     --volume=/home/ubuntu/CGP-Somatic-Docker-original/./datastore/launcher-ba5b9c2f-0b64-473b-8c9f-99e1153ad0de/inputs/d3ae586e-1251-470b-bbf8-f498e5895312/GRCh37d5_battenberg.tar.gz:/var/lib/cwl/stg4e55e3b3-46aa-4c4b-b4d6-3f2749498168/GRCh37d5_battenberg.tar.gz:ro \
</I>&gt;&gt;<i>     --volume=/tmp/tmp0NOg7v:/var/spool/cwl:rw \
</I>&gt;&gt;<i>     --volume=/home/ubuntu/CGP-Somatic-Docker-original/datastore/launcher-ba5b9c2f-0b64-473b-8c9f-99e1153ad0de/workingQriPMs:/tmp:rw \
</I>&gt;&gt;<i>     --workdir=/var/spool/cwl \
</I>&gt;&gt;<i>     --read-only=true \
</I>&gt;&gt;<i>     --user=1000 \
</I>&gt;&gt;<i>     --env=TMPDIR=/tmp \
</I>&gt;&gt;<i>     --env=HOME=/var/spool/cwl \
</I>&gt;&gt;<i>     quay.io/pancancer/pcawg-sanger-cgp-workflow:2.0.0-cwl1 &lt;x-<A HREF="msg://7/redir.aspx?REF=4xVT1d2OiypnFeCFXosdCvpxbtXo4eyJ-2eROBgvFKOSjIFQXvjTCAFodHRwOi8vcXVheS5pby9wYW5jYW5jZXIvcGNhd2ctc2FuZ2VyLWNncC13b3JrZmxvdzoyLjAuMC1jd2wx">msg://7/redir.aspx?REF=4xVT1d2OiypnFeCFXosdCvpxbtXo4eyJ-2eROBgvFKOSjIFQXvjTCAFodHRwOi8vcXVheS5pby9wYW5jYW5jZXIvcGNhd2ctc2FuZ2VyLWNncC13b3JrZmxvdzoyLjAuMC1jd2wx</A>&gt; \
</I>&gt;&gt;<i>     python \
</I>&gt;&gt;<i>     /home/seqware/CGP-Somatic-Docker/scripts/run_seqware_workflow.py \
</I>&gt;&gt;<i>     --tumor \
</I>&gt;&gt;<i>     /var/lib/cwl/stg2e1e9218-4ca7-4cbb-b474-dfdf6ac0b8a6/7875b5196f6b8b52847f99bf370aada0.bam \
</I>&gt;&gt;<i>     --normal \
</I>&gt;&gt;<i>     /var/lib/cwl/stge94eb32c-dc0f-4523-a88c-d1e00a322537/fdcb1bd7cffca69d15383ca9566c58e0.bam \
</I>&gt;&gt;<i>     --refFrom \
</I>&gt;&gt;<i>     /var/lib/cwl/stg218c30fe-3a28-4c3a-9803-253754dae462/GRCh37d5_CGP_refBundle.tar.gz \
</I>&gt;&gt;<i>     --bbFrom \
</I>&gt;&gt;<i>     /var/lib/cwl/stg4e55e3b3-46aa-4c4b-b4d6-3f2749498168/GRCh37d5_battenberg.tar.gz
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> The listing of the working directory (/var/spool/cwl) is as follows and does seem to include generated bas files:
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">ubuntu at sanger-retest</A>:/tmp/tmp0NOg7v$ ls -alhtr
</I>&gt;&gt;<i> total 92K
</I>&gt;&gt;<i> -rw-r--r--  1 ubuntu ubuntu 1.6K Oct 14 22:07 workflow.ini
</I>&gt;&gt;<i> -rw-r--r--  1 ubuntu ubuntu   28 Oct 14 22:07 .Rprofile
</I>&gt;&gt;<i> drwxr-xr-x  3 ubuntu root   4.0K Oct 14 22:07 .seqware
</I>&gt;&gt;<i> drwxr-xr-x  2 ubuntu ubuntu 4.0K Oct 14 22:07 ngsCounts
</I>&gt;&gt;<i> lrwxrwxrwx  1 ubuntu ubuntu   89 Oct 14 22:07 7875b5196f6b8b52847f99bf370aada0.bam -&gt; /var/lib/cwl/stg2e1e9218-4ca7-4cbb-b474-dfdf6ac0b8a6/7875b5196f6b8b52847f99bf370aada0.bam
</I>&gt;&gt;<i> lrwxrwxrwx  1 ubuntu ubuntu   93 Oct 14 22:07 fdcb1bd7cffca69d15383ca9566c58e0.bam.bai -&gt; /var/lib/cwl/stge94eb32c-dc0f-4523-a88c-d1e00a322537/fdcb1bd7cffca69d15383ca9566c58e0.bam.bai
</I>&gt;&gt;<i> lrwxrwxrwx  1 ubuntu ubuntu   89 Oct 14 22:07 fdcb1bd7cffca69d15383ca9566c58e0.bam -&gt; /var/lib/cwl/stge94eb32c-dc0f-4523-a88c-d1e00a322537/fdcb1bd7cffca69d15383ca9566c58e0.bam
</I>&gt;&gt;<i> lrwxrwxrwx  1 ubuntu ubuntu   93 Oct 14 22:07 7875b5196f6b8b52847f99bf370aada0.bam.bai -&gt; /var/lib/cwl/stg2e1e9218-4ca7-4cbb-b474-dfdf6ac0b8a6/7875b5196f6b8b52847f99bf370aada0.bam.bai
</I>&gt;&gt;<i> drwxr-xr-x  3 ubuntu ubuntu 4.0K Oct 14 22:07 1
</I>&gt;&gt;<i> drwxr-xr-x  2 ubuntu ubuntu 4.0K Oct 14 22:07 genotype
</I>&gt;&gt;<i> drwxr-xr-x  8 ubuntu ubuntu 4.0K Oct 14 22:19 reference_files
</I>&gt;&gt;<i> drwxr-xr-x  2 ubuntu ubuntu 4.0K Oct 14 22:19 genotype_b02b4bba-6e66-44fb-a48f-38c309aaaac5
</I>&gt;&gt;<i> -rw-r--r--  1 ubuntu ubuntu 2.4K Oct 14 22:19 b02b4bba-6e66-44fb-a48f-38c309aaaac5.genotype.tar.gz
</I>&gt;&gt;<i> -rw-r--r--  1 ubuntu ubuntu   33 Oct 14 22:19 b02b4bba-6e66-44fb-a48f-38c309aaaac5.genotype.tar.gz.md5
</I>&gt;&gt;<i> -rw-r--r--  1 ubuntu ubuntu 1.5K Oct 14 22:58 fdcb1bd7cffca69d15383ca9566c58e0.bam.bas
</I>&gt;&gt;<i> -rw-r--r--  1 ubuntu ubuntu 2.3K Oct 14 23:17 7875b5196f6b8b52847f99bf370aada0.bam.bas
</I>&gt;&gt;<i> -rw-r--r--  1 ubuntu ubuntu   33 Oct 15 08:23 b02b4bba-6e66-44fb-a48f-38c309aaaac5.csc_0-0-0.20161014.somatic.genotype.tar.gz.md5
</I>&gt;&gt;<i> -rw-r--r--  1 ubuntu ubuntu 2.4K Oct 15 08:23 b02b4bba-6e66-44fb-a48f-38c309aaaac5.csc_0-0-0.20161014.somatic.genotype.tar.gz
</I>&gt;&gt;<i> drwxr-xr-x  2 ubuntu ubuntu 4.0K Oct 15 10:02 timings
</I>&gt;&gt;<i> drwxr-xr-x  5 ubuntu ubuntu 4.0K Oct 15 10:02 0
</I>&gt;&gt;<i> drwxr-xr-x  3 ubuntu ubuntu 4.0K Oct 15 10:06 bbCounts
</I>&gt;&gt;<i> drwx------ 11 ubuntu ubuntu 4.0K Oct 15 20:28 .
</I>&gt;&gt;<i> drwxrwxrwt 21 root   root   4.0K Oct 15 20:29 ..
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> The full output of the failing script is:
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Errors from command: /usr/bin/perl /opt/wtsi-cgp/bin/pindel_input_gen.pl -b /var/lib/cwl/stge94eb32c-dc0f-4523-a88c-d1e00a322537/fdcb1bd7cffca69d15383ca9566c58e0.bam -o /var/spool/cwl/0/pindel/tmpPindel/8c0354eb-6a3e-4a98-b41c-f8add599884c -t 4 -e /var/spool/cwl/reference_files/brass/ucscHiDepth_0.01_mrg1000_no_exon_coreChrs.bed.gz
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Unknown sort order field: unknown
</I>&gt;&gt;<i> Collated 500000 readpairs (in 6 sec.)
</I>&gt;&gt;<i> [V] 1   34.4825MB/s     133279
</I>&gt;&gt;<i> Thread Worker 1: started
</I>&gt;&gt;<i> Thread 1 terminated abnormally: Failed to get insert size for readgroup: 'CRUK-CI:LP6005333-DNA_C03''
</I>&gt;&gt;<i> Collated 500000 readpairs (in 4 sec.)
</I>&gt;&gt;<i> [V] 2   39.9836MB/s     154626
</I>&gt;&gt;<i> Thread Worker 2: started
</I>&gt;&gt;<i> Thread 2 terminated abnormally: Failed to get insert size for readgroup: 'CRUK-CI:LP6005333-DNA_C03''
</I>&gt;&gt;<i> Collated 500000 readpairs (in 4 sec.)
</I>&gt;&gt;<i> Thread Worker 3: started
</I>&gt;&gt;<i> Thread 3 terminated abnormally: Failed to get insert size for readgroup: 'CRUK-CI:LP6005333-DNA_C03'
</I>&gt;&gt;<i> [V] 3   42.4188MB/s     164102
</I>&gt;&gt;<i> Collated 500000 readpairs (in 4 sec.)
</I>&gt;&gt;<i> Thread Worker 4: started
</I>&gt;&gt;<i> Thread 4 terminated abnormally: Failed to get insert size for readgroup: 'CRUK-CI:LP6005333-DNA_C03''
</I>&gt;&gt;<i> [V] 4   43.7799MB/s     169368
</I>&gt;&gt;<i> Collated 500000 readpairs (in 4 sec.)
</I>&gt;&gt;<i> An error occurred while running:
</I>&gt;&gt;<i>         /opt/wtsi-cgp/bin/bamcollate2 outputformat=sam colsbs=268435456 collate=1 classes=F,F2 exclude=DUP,SECONDARY,QCFAIL,SUPPLEMENTARY T=/var/spool/cwl/0/pindel/tmpPindel/8c0354eb-6a3e-4a98-b41c-f8add599884c/tmp1kNw/collate_tmp filename=/var/lib/cwl/stge94eb32c-dc0f-4523-a88c-d1e00a322537/fdcb1bd7cffca69d15383ca9566c58e0.bam
</I>&gt;&gt;<i> ERROR: Converter thread error: Failed to get insert size for readgroup: 'CRUK-CI:LP6005333-DNA_C03''
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Perl exited with active threads:
</I>&gt;&gt;<i>         0 running and unjoined
</I>&gt;&gt;<i>         3 finished and unjoined
</I>&gt;&gt;<i>         0 running and detached
</I>&gt;&gt;<i> Thread 2 terminated abnormally: main=HASH(0x50044b0) at /opt/wtsi-cgp/lib/perl5/PCAP/Threaded.pm line 115.
</I>&gt;&gt;<i> Thread error: &quot;/usr/bin/perl /opt/wtsi-cgp/bin/pindel_input_gen.pl -b /var/lib/cwl/stge94eb32c-dc0f-4523-a88c-d1e00a322537/fdcb1bd7cffca69d15383ca9566c58e0.bam -o /var/spool/cwl/0/pindel/tmpPindel/8c0354eb-6a3e-4a98-b41c-f8add599884c -t 4 -e /var/spool/cwl/reference_files/brass/ucscHiDepth_0.01_mrg1000_no_exon_coreChrs.bed.gz&quot; unexpectedly returned exit value 29 at (eval 410) line 13 thread 2.
</I>&gt;&gt;<i>  at /opt/wtsi-cgp/lib/perl5/PCAP/Threaded.pm line 190
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Command exited with non-zero status 25
</I>&gt;&gt;<i> 10.23user 8.79system 0:32.33elapsed 58%CPU (0avgtext+0avgdata 10819936maxresident)k
</I>&gt;&gt;<i> 1184inputs+8552outputs (2major+711948minor)pagefaults 0swaps 
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Please let me know if any of the files on that host would be useful to debug this.
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> From: <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">mikisvaz at gmail.com</A> &lt;x-<A HREF="msg://7/redir.aspx?REF=HLi2gsYnkByZIrxw9tkgQ3smNIdi5933-xaaJSjcikiSjIFQXvjTCAFtYWlsdG86bWlraXN2YXpAZ21haWwuY29t">msg://7/redir.aspx?REF=HLi2gsYnkByZIrxw9tkgQ3smNIdi5933-xaaJSjcikiSjIFQXvjTCAFtYWlsdG86bWlraXN2YXpAZ21haWwuY29t</A>&gt; [<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">mikisvaz at gmail.com</A> &lt;x-<A HREF="msg://7/redir.aspx?REF=HLi2gsYnkByZIrxw9tkgQ3smNIdi5933-xaaJSjcikiSjIFQXvjTCAFtYWlsdG86bWlraXN2YXpAZ21haWwuY29t">msg://7/redir.aspx?REF=HLi2gsYnkByZIrxw9tkgQ3smNIdi5933-xaaJSjcikiSjIFQXvjTCAFtYWlsdG86bWlraXN2YXpAZ21haWwuY29t</A>&gt;] on behalf of Miguel Vazquez [<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">miguel.vazquez at cnio.es</A> &lt;x-<A HREF="msg://7/redir.aspx?REF=82FKQEdg1KYeV7fUIdfKnUibR7K5SoE50-vpnDSw3VqSjIFQXvjTCAFtYWlsdG86bWlndWVsLnZhenF1ZXpAY25pby5lcw..">msg://7/redir.aspx?REF=82FKQEdg1KYeV7fUIdfKnUibR7K5SoE50-vpnDSw3VqSjIFQXvjTCAFtYWlsdG86bWlndWVsLnZhenF1ZXpAY25pby5lcw..</A>&gt;]
</I>&gt;&gt;<i> Sent: October 15, 2016 2:52 AM
</I>&gt;&gt;<i> To: Denis Yuen
</I>&gt;&gt;<i> Cc: Adam Struck; Keiran Raine; <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A> &lt;x-<A HREF="msg://7/redir.aspx?REF=kid5T9Srz1MPcWdg34ViTlJk22Q3G-C_70CPWVlxSX2SjIFQXvjTCAFtYWlsdG86ZG9ja3Rlc3RlcnNAbGlzdHMuaWNnYy5vcmc.">msg://7/redir.aspx?REF=kid5T9Srz1MPcWdg34ViTlJk22Q3G-C_70CPWVlxSX2SjIFQXvjTCAFtYWlsdG86ZG9ja3Rlc3RlcnNAbGlzdHMuaWNnYy5vcmc.</A>&gt;
</I>&gt;&gt;<i> Subject: Re: [DOCKTESTERS] [PAWG-TECH] Draft agenda for PCAWG-TECH teleconference
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Just to clarify Denis, the BAS file was not present in the download of the donor data, while it was present in the download of the test data. That is as much as I observed, and this matched Keiran comment that a missing BAS file was consistent with the pindel error.I have no idea of what the workflow was doing so as far as I know the BAS could have been created correctly and the error was something else. 
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Best regards
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> On Sat, Oct 15, 2016 at 12:20 AM, Denis Yuen &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">Denis.Yuen at oicr.on.ca</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;&gt; wrote:
</I>&gt;&gt;<i> Hi,
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Adam, to summarise:
</I>&gt;&gt;<i> My observations seem to match yours, the bas file and input bams are generated inside the Docker container with the test data. However, Miguel has observed that something else seems to be happening with DO50311 that looks like the bas file being missing. I'm currently running that donor to see if I can extract more information and determine what is occurring. 
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> From: Adam Struck [<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">strucka at ohsu.edu</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;]
</I>&gt;&gt;<i> Sent: October 14, 2016 6:10 PM
</I>&gt;&gt;<i> To: Denis Yuen; Keiran Raine
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Cc: <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;&gt;<i> Subject: Re: [DOCKTESTERS] [PAWG-TECH] Draft agenda for PCAWG-TECH teleconference
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Hi Denis,
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> Sorry, to chime in late.  The bas file and input BAMs should be getting colocalized already (see below). Where are these files ending up when you run the workflow?
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> Inputs are symlinked to the OUTDIR.
</I>&gt;&gt;<i> <A HREF="https://github.com/adamstruck/CGP-Somatic-Docker/blob/develop/src/main/java/io/seqware/pancancer/CgpSomaticCore.java#L1035-L1039">https://github.com/adamstruck/CGP-Somatic-Docker/blob/develop/src/main/java/io/seqware/pancancer/CgpSomaticCore.java#L1035-L1039</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> Bas files are written to OUTDIR
</I>&gt;&gt;<i> <A HREF="https://github.com/adamstruck/CGP-Somatic-Docker/blob/develop/src/main/java/io/seqware/pancancer/CgpSomaticCore.java#L769-L780">https://github.com/adamstruck/CGP-Somatic-Docker/blob/develop/src/main/java/io/seqware/pancancer/CgpSomaticCore.java#L769-L780</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> I have now run 25 donors worth of data (from PRAD-UK) through this workflow using the WDL descriptor and the cromwell engine on the CCC platform without an issue.
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> -Adam 
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> From: &lt;docktesters-bounces+strucka=<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">ohsu.edu at lists.icgc.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;&gt; on behalf of Denis Yuen &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">Denis.Yuen at oicr.on.ca</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;&gt;
</I>&gt;&gt;<i> Date: Friday, October 14, 2016 at 2:55 PM
</I>&gt;&gt;<i> To: Keiran Raine &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;&gt;
</I>&gt;&gt;<i> Cc: &quot;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;&quot; &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;&gt;
</I>&gt;&gt;<i> Subject: Re: [DOCKTESTERS] [PAWG-TECH] Draft agenda for PCAWG-TECH teleconference
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> Hi,
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Just as a heads-up for the end-of-week in this thread. 
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> &gt; RUN cpanm --mirror <A HREF="http://cpan.metacpan.org">http://cpan.metacpan.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt; -l $OPT File::ShareDir File::ShareDir::Install Bio::Root::<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">Version at 1.006924</A> Const::Fast Graph &amp;&amp; \
</I>&gt;&gt;<i>      rm -rf ~/.cpanm
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> This got me on the right track, I actually needed the following syntax
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> RUN cpanm --mirror <A HREF="https://cpan.metacpan.org">https://cpan.metacpan.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt; -l $OPT File::ShareDir File::ShareDir::Install CJFIELDS/BioPerl-1.6.924.tar.gz Const::Fast Graph &amp;&amp; \
</I>&gt;&gt;<i>      rm -rf ~/.cpanm
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> However, it looks like making the suggested change breaks the workflow when attempting to run with the test data. In short, the bas file is definitely being generated inside the Docker container. Moving it to the suggested location breaks the workflow later. 
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> I'm currently attempting to run the donor DO50311 to see if I can get more insight into what is going on.
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> From: Keiran Raine [<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;]
</I>&gt;&gt;<i> Sent: October 12, 2016 2:09 PM
</I>&gt;&gt;<i> To: Denis Yuen
</I>&gt;&gt;<i> Cc: <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;&gt;<i> Subject: Re: [DOCKTESTERS] [PAWG-TECH] Draft agenda for PCAWG-TECH teleconference
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Hi Denis,
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> You've hit an issue that only has occurred in the last few weeks for us also.
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> BioPerl released a new version (first in ~20 months) that split the repository moving a whole section into a different package.
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> The fix would be to force the first install of BioPerl to a specific version.  Modify line 25/26 of the Dockerfile from:
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> RUN cpanm --mirror <A HREF="http://cpan.metacpan.org">http://cpan.metacpan.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt; -l $OPT File::ShareDir File::ShareDir::Install Bio::Root::Version Const::Fast Graph &amp;&amp; \
</I>&gt;&gt;<i>      rm -rf ~/.cpanm
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> to
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> RUN cpanm --mirror <A HREF="http://cpan.metacpan.org">http://cpan.metacpan.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt; -l $OPT File::ShareDir File::ShareDir::Install Bio::Root::<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">Version at 1.006924</A> Const::Fast Graph &amp;&amp; \
</I>&gt;&gt;<i>      rm -rf ~/.cpanm
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> Thankfully something I could identify immediately.
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> Regards,
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Keiran Raine
</I>&gt;&gt;<i> Principal Bioinformatician
</I>&gt;&gt;<i> Cancer Genome Project
</I>&gt;&gt;<i> Wellcome Trust Sanger Institute
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;&gt;<i> Tel:+44 (0)1223 834244 Ext: 4983 &lt;tel:%2B44%20%280%291223%20834244%20Ext%3A%204983&gt;
</I>&gt;&gt;<i> Office: H104
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> On 12 Oct 2016, at 17:04, Denis Yuen &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">Denis.Yuen at oicr.on.ca</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;&gt; wrote:
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> Hi, 
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Keiran, I'm having trouble rebuilding the Sanger docker container in what I think is an unrelated section.
</I>&gt;&gt;<i> Has anything changed about the build dependencies (for example, if there is a floating version that changed over time)?
</I>&gt;&gt;<i> I'm attaching the build log from the Dockerfile and the log from inside the container.
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> From: docktesters-bounces+denis.yuen=<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">oicr.on.ca at lists.icgc.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt; [docktesters-bounces+denis.yuen=<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">oicr.on.ca at lists.icgc.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;] on behalf of Denis Yuen [<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">Denis.Yuen at oicr.on.ca</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;]
</I>&gt;&gt;<i> Sent: October 12, 2016 10:59 AM
</I>&gt;&gt;<i> To: Keiran Raine
</I>&gt;&gt;<i> Cc: <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;&gt;<i> Subject: Re: [DOCKTESTERS] [PAWG-TECH] Draft agenda for PCAWG-TECH teleconference
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Hi,
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> While that would have been a good explanation, unfortunately, it doesn't seem to be the case. 
</I>&gt;&gt;<i> In the CWL file ( <A HREF="https://github.com/ICGC-TCGA-PanCancer/CGP-Somatic-Docker/blob/develop/Dockstore.cwl">https://github.com/ICGC-TCGA-PanCancer/CGP-Somatic-Docker/blob/develop/Dockstore.cwl</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt; ) , the bam files are described like 
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> inputs:
</I>&gt;&gt;<i>   tumor:
</I>&gt;&gt;<i>     type: File
</I>&gt;&gt;<i>     inputBinding:
</I>&gt;&gt;<i>       position: 1
</I>&gt;&gt;<i>       prefix: --tumor
</I>&gt;&gt;<i>     secondaryFiles:
</I>&gt;&gt;<i>     - .bai
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i>   refFrom:
</I>&gt;&gt;<i>     type: File
</I>&gt;&gt;<i>     inputBinding:
</I>&gt;&gt;<i>       position: 3
</I>&gt;&gt;<i>       prefix: --refFrom
</I>&gt;&gt;<i>   bbFrom:
</I>&gt;&gt;<i>     type: File
</I>&gt;&gt;<i>     inputBinding:
</I>&gt;&gt;<i>       position: 4
</I>&gt;&gt;<i>       prefix: --bbFrom
</I>&gt;&gt;<i>   normal:
</I>&gt;&gt;<i>     type: File
</I>&gt;&gt;<i>     inputBinding:
</I>&gt;&gt;<i>       position: 2
</I>&gt;&gt;<i>       prefix: --normal
</I>&gt;&gt;<i>     secondaryFiles:
</I>&gt;&gt;<i>     - .bai
</I>&gt;&gt;<i> The type of File (as opposed to directory) means that while the bam and bai files are individually mounted into the docker container while it runs, the bas files never were. If Miguel has the &quot;docker run&quot; output from the run (should just be in the stdout of the run), we should be able to verify this by looking at what gets mounted into the container at runtime. 
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> From: Keiran Raine [<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;]
</I>&gt;&gt;<i> Sent: October 12, 2016 10:49 AM
</I>&gt;&gt;<i> To: Denis Yuen
</I>&gt;&gt;<i> Cc: Miguel Vazquez; <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;&gt;<i> Subject: Re: [DOCKTESTERS] [PAWG-TECH] Draft agenda for PCAWG-TECH teleconference
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Hi Denis, 
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> I expect when you unpack the the test data the BAS files exist in the archive in that area so the fact it runs the out of that step to a different location isn't detected.
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Keiran Raine
</I>&gt;&gt;<i> Principal Bioinformatician
</I>&gt;&gt;<i> Cancer Genome Project
</I>&gt;&gt;<i> Wellcome Trust Sanger Institute
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;&gt;<i> Tel:+44 (0)1223 834244 Ext: 4983 &lt;tel:%2B44%20%280%291223%20834244%20Ext%3A%204983&gt;
</I>&gt;&gt;<i> Office: H104
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> On 12 Oct 2016, at 15:36, Denis Yuen &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">Denis.Yuen at oicr.on.ca</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;&gt; wrote:
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> Hi,
</I>&gt;&gt;<i> I can make the modification, I'll run it through the test data and that should finish in roughly a day. 
</I>&gt;&gt;<i> In the meantime though, I am puzzled. Why would an issue like this affect a donor dataset, but not the test data?
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> From: docktesters-bounces+denis.yuen=<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">oicr.on.ca at lists.icgc.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt; [docktesters-bounces+denis.yuen=<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">oicr.on.ca at lists.icgc.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;] on behalf of Keiran Raine [<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;]
</I>&gt;&gt;<i> Sent: October 12, 2016 6:16 AM
</I>&gt;&gt;<i> To: Miguel Vazquez
</I>&gt;&gt;<i> Cc: <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;&gt;<i> Subject: Re: [DOCKTESTERS] [PAWG-TECH] Draft agenda for PCAWG-TECH teleconference
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Hi, 
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> This is assuming that it is possible to write to the location the BAM are in.
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> I think Denis would be best placed to make the minor modification as I don't know the process they are using for build and deploy of the images (I made modifications and then handed over for CWL).
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> Regards,
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Keiran Raine
</I>&gt;&gt;<i> Principal Bioinformatician
</I>&gt;&gt;<i> Cancer Genome Project
</I>&gt;&gt;<i> Wellcome Trust Sanger Institute
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;&gt;<i> Tel:+44 (0)1223 834244 Ext: 4983 &lt;tel:%2B44%20%280%291223%20834244%20Ext%3A%204983&gt;
</I>&gt;&gt;<i> Office: H104
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> On 12 Oct 2016, at 10:37, Miguel Vazquez &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">miguel.vazquez at cnio.es</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;&gt; wrote:
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> [The rest of the list where out of the loop for this part of the conversation, I'm putting them back in. In short, the Sanger pipeline produces the BAS file but not co-located with the BAM]
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Hi Keiran,
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Would it be possible then to change this and try again? what needs to happen? I guess you'll need to change the code and a new docker image be produced. Would this be our best alternative? 
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> Miguel
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> On Tue, Oct 11, 2016 at 4:07 PM, Keiran Raine &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;&gt; wrote:
</I>&gt;&gt;<i> In the original version we didn't do this step, if we have write access it can be made to do that
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Keiran Raine
</I>&gt;&gt;<i> Principal Bioinformatician
</I>&gt;&gt;<i> Cancer Genome Project
</I>&gt;&gt;<i> Wellcome Trust Sanger Institute
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;&gt;<i> Tel:+44 (0)1223 834244 Ext: 4983 &lt;tel:%2B44%20%280%291223%20834244%20Ext%3A%204983&gt;
</I>&gt;&gt;<i> Office: H104
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> On 11 Oct 2016, at 13:49, Miguel Vazquez &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">miguel.vazquez at cnio.es</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;&gt; wrote:
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> Hi Keiran,
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> If the BAS and BAM files need to be collocated, why is it not created next to the BAM file?
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Would it not be better if it read
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> private Job basFileBaseJob(int tumourCount, String sampleBam, String process, int index) { 
</I>&gt;&gt;<i> Job thisJob = prepTimedJob(tumourCount, &quot;basFileGenerate&quot;, process, index);
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> File f = new File(sampleBam);
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> thisJob.getCommand()
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> .addArgument(getWorkflowBaseDir()+ &quot;/bin/wrapper.sh&quot;)
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> .addArgument(installBase)
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> .addArgument(&quot;bam_stats&quot;)
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> .addArgument(&quot;-i &quot; + sampleBam)
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> .addArgument(&quot;-o &quot; + sampleBam + &quot;.bas&quot;)
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> ;
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> return thisJob;
</I>&gt;&gt;<i> }
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> Best
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Miguel
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> On Tue, Oct 11, 2016 at 2:44 PM, Keiran Raine &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;&gt; wrote:
</I>&gt;&gt;<i> Relevant section of code: 
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> <A HREF="https://github.com/ICGC-TCGA-PanCancer/CGP-Somatic-Docker/blob/develop/src/main/java/io/seqware/pancancer/CgpSomaticCore.java#L769-L780">https://github.com/ICGC-TCGA-PanCancer/CGP-Somatic-Docker/blob/develop/src/main/java/io/seqware/pancancer/CgpSomaticCore.java#L769-L780</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Keiran Raine
</I>&gt;&gt;<i> Principal Bioinformatician
</I>&gt;&gt;<i> Cancer Genome Project
</I>&gt;&gt;<i> Wellcome Trust Sanger Institute
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;&gt;<i> Tel:+44 (0)1223 834244 Ext: 4983 &lt;tel:%2B44%20%280%291223%20834244%20Ext%3A%204983&gt;
</I>&gt;&gt;<i> Office: H104
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> On 11 Oct 2016, at 13:40, Keiran Raine &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;&gt; wrote:
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> Hi, 
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> There is a step generating the BAS files:
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> [2016/10/10 07:28:37] |  Running command: bash /datastore/oozie-6599f0b9-8af7-44ca-a608-03c5bbc159c6/generated-scripts/s58_basFileGenerate_control_11-runner.sh
</I>&gt;&gt;<i> [2016/10/10 07:28:37] |  Running command: bash /datastore/oozie-6599f0b9-8af7-44ca-a608-03c5bbc159c6/generated-scripts/s58_basFileGenerate_tumours_12-runner.sh
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> But if the BAM files and BAS aren't co-located then you have a problem.  You could symlink the BAM files into the work space and have all tools work from that path instead, deleting the symlinks at the end.
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> This is one of the changes we had to implement differently as the BAS file data was being held in the GNOS xml data structures during the initial processing.  Moving to this means that any BAM input is sufficient.
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> Hope this is easier to solve now,
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Keiran Raine
</I>&gt;&gt;<i> Principal Bioinformatician
</I>&gt;&gt;<i> Cancer Genome Project
</I>&gt;&gt;<i> Wellcome Trust Sanger Institute
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">kr2 at sanger.ac.uk</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;&gt;<i> Tel:+44 (0)1223 834244 Ext: 4983 &lt;tel:%2B44%20%280%291223%20834244%20Ext%3A%204983&gt;
</I>&gt;&gt;<i> Office: H104
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> On 11 Oct 2016, at 13:31, Miguel Vazquez &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">miguel.vazquez at cnio.es</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;&gt; wrote:
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> Keiran,
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Its downloading the files still but in fact it does not seem to download any BAS file. Could you please educate me a bit on what are these and how I can create them?
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Best
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Miguel
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> On Tue, Oct 11, 2016 at 2:22 PM, Miguel Vazquez &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">miguel.vazquez at cnio.es</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;&gt; wrote:
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> 4. It looks like the *_pindel_input_* steps run for only 22-23 seconds which could indicate a problem with either the headers or the absence of the BAS file from the expected location.
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> I think that you just revealed the problem. There is in fact no BAS files only BAM and BAI. There where BAS files for the test data HCC1143 which is the one that in fact work. It seems like BAS files are not gathered by gnos, could that be? or that my script fails to copy them. I'll try to gather a different sample with my client and check. 
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Not knowing a thing about these files explains why I didn't notice. I'll get back to you when I know more.
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Best
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> Miguel
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> -- The Wellcome Trust Sanger Institute is operated by Genome Research Limited, a charity registered in England with number 1021457 and a company registered in England with number 2742969, whose registered office is 215 Euston Road, London, NW1 2BE. 
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> -- The Wellcome Trust Sanger Institute is operated by Genome Research Limited, a charity registered in England with number 1021457 and a company registered in England with number 2742969, whose registered office is 215 Euston Road, London, NW1 2BE. 
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> -- The Wellcome Trust Sanger Institute is operated by Genome Research Limited, a charity registered in England with number 1021457 and a company registered in England with number 2742969, whose registered office is 215 Euston Road, London, NW1 2BE.
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> -- The Wellcome Trust Sanger Institute is operated by Genome Research Limited, a charity registered in England with number 1021457 and a company registered in England with number 2742969, whose registered office is 215 Euston Road, London, NW1 2BE. 
</I>&gt;&gt;<i> &lt;build.log&gt;&lt;outer.build.log&gt;
</I>&gt;&gt;<i>  
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> -- The Wellcome Trust Sanger Institute is operated by Genome Research Limited, a charity registered in England with number 1021457 and a company registered in England with number 2742969, whose registered office is 215 Euston Road, London, NW1 2BE.
</I>&gt;&gt;<i> 
</I>&gt;&gt;<i> _______________________________________________
</I>&gt;&gt;<i> docktesters mailing list
</I>&gt;&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">docktesters at lists.icgc.org</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">https://lists.icgc.org/mailman/listinfo/docktesters</A> &lt;x-<A HREF="msg://7/UrlBlockedError.aspx">msg://7/UrlBlockedError.aspx</A>&gt;
</I>&gt;<i> 
</I>&gt;<i> -- The Wellcome Trust Sanger Institute is operated by Genome Research Limited, a charity registered in England with number 1021457 and a company registered in England with number 2742969, whose registered office is 215 Euston Road, London, NW1 2BE.
</I>



-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE. 

-------------- next part --------------
An HTML attachment was scrubbed...
URL: &lt;<A HREF="https://lists.icgc.org/mailman/private/docktesters/attachments/20161020/c9182da9/attachment-0001.html">https://lists.icgc.org/mailman/private/docktesters/attachments/20161020/c9182da9/attachment-0001.html</A>&gt;
</PRE>





<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="000130.html">[DOCKTESTERS] Sanger symlinked input file issue
</A></li>
	<LI>Next message: <A HREF="000133.html">[DOCKTESTERS] Broad PCAWG Tokens definition?
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#131">[ date ]</a>
              <a href="thread.html#131">[ thread ]</a>
              <a href="subject.html#131">[ subject ]</a>
              <a href="author.html#131">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="https://lists.icgc.org/mailman/listinfo/docktesters">More information about the docktesters
mailing list</a><br>
</body></html>
