<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [DOCKTESTERS]  BWA-Mem validation of DO35937. 93% matches, 3.6% miss-matches, and 3.2% soft-matches
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:docktesters%40lists.icgc.org?Subject=Re%3A%20%5BDOCKTESTERS%5D%20%20BWA-Mem%20validation%20of%20DO35937.%2093%25%20matches%2C%0A%203.6%25%20miss-matches%2C%20and%203.2%25%20soft-matches&In-Reply-To=%3CCA%2Bdx8mb2SJqrA-xYf_SLGuooD2mG5byyu4t%2BUBywjbXMKxXBvQ%40mail.gmail.com%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="000236.html">
   <LINK REL="Next"  HREF="000238.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[DOCKTESTERS]  BWA-Mem validation of DO35937. 93% matches, 3.6% miss-matches, and 3.2% soft-matches</H1>
    <B>Miguel Vazquez</B> 
    <A HREF="mailto:docktesters%40lists.icgc.org?Subject=Re%3A%20%5BDOCKTESTERS%5D%20%20BWA-Mem%20validation%20of%20DO35937.%2093%25%20matches%2C%0A%203.6%25%20miss-matches%2C%20and%203.2%25%20soft-matches&In-Reply-To=%3CCA%2Bdx8mb2SJqrA-xYf_SLGuooD2mG5byyu4t%2BUBywjbXMKxXBvQ%40mail.gmail.com%3E"
       TITLE="[DOCKTESTERS]  BWA-Mem validation of DO35937. 93% matches, 3.6% miss-matches, and 3.2% soft-matches">miguel.vazquez at cnio.es
       </A><BR>
    <I>Sun Feb 19 07:43:11 EST 2017</I>
    <P><UL>
        <LI>Previous message: <A HREF="000236.html">[DOCKTESTERS] BWA-Mem validation of HCC1143. 95% matches, 3.6% miss-matches, and 1.3% soft-matches
</A></li>
        <LI>Next message: <A HREF="000238.html">[DOCKTESTERS] Questions about BiasFilter
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#237">[ date ]</a>
              <a href="thread.html#237">[ thread ]</a>
              <a href="subject.html#237">[ subject ]</a>
              <a href="author.html#237">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>Dear all,

Great news! The BWA-Mem test on a real PCAWG donor succeed in running;
achieving an overlap with the original BAM alignment similar to the
HCC1143 test. The numbers are:

Lines: 1708047647
Matches: 1589172843
Misses: 62726130
Soft: 56148674

Which mean 93% matches, 3.6% miss-matches, and 3.2% soft-matches. Compared
to the HCC1143 test there are a few percentage points in matches that turn
into soft-matches (95% and 1.3% to 93% and 3.2%), but the ratio of misses
is very close 3.6%.

I'm running this test on a second donor.

Best regards

Miguel

On Tue, Feb 14, 2017 at 3:30 PM, Miguel Vazquez &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">miguel.vazquez at cnio.es</A>&gt;
wrote:

&gt;<i> Dear colleagues,
</I>&gt;<i>
</I>&gt;<i> I'm very happy to say that the BWA-Mem pipeline finished for the HCC1143
</I>&gt;<i> data.
</I>&gt;<i>
</I>&gt;<i> I think what solved the problem was setting the headers to the unaligned
</I>&gt;<i> BAM files. I'm currently trying it out with the DO35937 donor, but its too
</I>&gt;<i> early to say if its working or not.
</I>&gt;<i>
</I>&gt;<i> To compare BAM files I've followed some advice that I found on the
</I>&gt;<i> internet <A HREF="https://www.biostars.org/p/166221/.">https://www.biostars.org/p/166221/.</A> I will detail them a bit
</I>&gt;<i> below because I would like some advice as to how appropriate the approach
</I>&gt;<i> is, but first here are the numbers:
</I>&gt;<i>
</I>&gt;<i> *Lines*: 74264390
</I>&gt;<i> *Matches*: 70565742
</I>&gt;<i> *Misses*: 2693687
</I>&gt;<i> *Soft*: 1004961
</I>&gt;<i>
</I>&gt;<i>
</I>&gt;<i> Which means *95% matches, 3.6% miss-matches, and 1.3% soft-matches*.
</I>&gt;<i> Matches are when the chromosome and position are the same, soft-matches are
</I>&gt;<i> when they are not the same but the position from one of the alignments is
</I>&gt;<i> included in the list of alternative positions for the other alignment (e.g
</I>&gt;<i> XA:Z:15,-102516528,76M,0), and misses are the rest.
</I>&gt;<i>
</I>&gt;<i> Here is the detailed process from the start. The comparison script is here
</I>&gt;<i> <A HREF="https://github.com/mikisvaz/PCAWG-Docker-Test/blob/master/">https://github.com/mikisvaz/PCAWG-Docker-Test/blob/master/</A>
</I>&gt;<i> bin/compare_bwa_bam.sh
</I>&gt;<i>
</I>&gt;<i> 1) Un-align tumor and normal BAM files, retaining the original aligned BAM
</I>&gt;<i> files
</I>&gt;<i> 2) Run BWA-Mem wich produces a file called HCC1143.merged_output.bam with
</I>&gt;<i> alignments from both tumor and normal
</I>&gt;<i> 3) use samtools to extract the entries, limited for the first in pair (?),
</I>&gt;<i> cut the read-name, chromosome, position (??) and extra information (for
</I>&gt;<i> additional alignments) and sort them. We do this for the original files and
</I>&gt;<i> for the BWA-Mem merged_output file, but separating tumor and normal entries
</I>&gt;<i> (marked with the codes 'tumor' and 'normal', I believe from the headers I
</I>&gt;<i> set when un-aligning them)
</I>&gt;<i> 4) join the lines by read-name, separately for the tumor and normal pairs
</I>&gt;<i> of files, and check for matches
</I>&gt;<i>
</I>&gt;<i> I've two questions:
</I>&gt;<i> (?) Is it OK to select only the first in pair, its what the guy in the
</I>&gt;<i> example did, and it did simplify the code without repeated read-names
</I>&gt;<i> (??) I guess its OK to only check chromosome and position, the cigar would
</I>&gt;<i> be necessarily the same.
</I>&gt;<i>
</I>&gt;<i> Best regards
</I>&gt;<i>
</I>&gt;<i> Miguel
</I>&gt;<i>
</I>&gt;<i> On Mon, Jan 16, 2017 at 3:24 PM, Miguel Vazquez &lt;<A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">miguel.vazquez at cnio.es</A>&gt;
</I>&gt;<i> wrote:
</I>&gt;<i>
</I>&gt;&gt;<i> Dear all,
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Let me summarize the status of the testing for Sanger and DKFZ. The
</I>&gt;&gt;<i> validation has been run for two donors for each workflow: DO50311 DO52140
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Sanger:
</I>&gt;&gt;<i> ----------
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Sanger call only somatic variants. The results are *identical for Indels
</I>&gt;&gt;<i> and SVs* but *almost identical for SNV.MNV and CNV*. The discrepancies
</I>&gt;&gt;<i> are reproducible (on the same machine at least), i.e. the same are found
</I>&gt;&gt;<i> after running the workflow a second time.
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> DKFZ:
</I>&gt;&gt;<i> ---------
</I>&gt;&gt;<i> DKFZ cals somatic and germline variants, except germline CNVs. For both
</I>&gt;&gt;<i> germline and somatic variants the results are *identical for SNV.MNV and
</I>&gt;&gt;<i> Indels* but with *large discrepancies for SV and CNV*.
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Kortine Kleinheinz and Joachim Weischenfeldt are in the process of
</I>&gt;&gt;<i> investigating this issue I believe.
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> BWA-Mem failed for me and has also failed for Denis Yuen and Jonas
</I>&gt;&gt;<i> Demeulemeester. Denis I believe is investigating this problem further. I
</I>&gt;&gt;<i> haven't had the chance to investigate this much myself.
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Best
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Miguel
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> ---------------------
</I>&gt;&gt;<i> RESULTS
</I>&gt;&gt;<i> ---------------------
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> <A HREF="https://lists.icgc.org/mailman/listinfo/docktesters">ubuntu at ip-10-253-35-14</A>:~/DockerTest-Miguel$ cat results.txt
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.snv.mnv for DO50311 using DKFZ
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 51087
</I>&gt;&gt;<i> Extra: 0
</I>&gt;&gt;<i> Missing: 0
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.indel for DO50311 using DKFZ
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 26469
</I>&gt;&gt;<i> Extra: 0
</I>&gt;&gt;<i> Missing: 0
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.sv for DO50311 using DKFZ
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 231
</I>&gt;&gt;<i> Extra: 44
</I>&gt;&gt;<i>     - Example: 10:20596800:N:&lt;TRA&gt;,10:5606682
</I>&gt;&gt;<i> 1:N:&lt;TRA&gt;,11:16776092:N:&lt;TRA&gt;
</I>&gt;&gt;<i> Missing: 48
</I>&gt;&gt;<i>     - Example: 10:119704959:N:&lt;INV&gt;,10:131163
</I>&gt;&gt;<i> 22:N:&lt;TRA&gt;,10:47063485:N:&lt;TRA&gt;
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.cnv for DO50311 using DKFZ
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 731
</I>&gt;&gt;<i> Extra: 213
</I>&gt;&gt;<i>     - Example: 10:132510034:N:&lt;DEL&gt;,10:205968
</I>&gt;&gt;<i> 01:N:&lt;NEUTRAL&gt;,10:47674883:N:&lt;NEUTRAL&gt;
</I>&gt;&gt;<i> Missing: 190
</I>&gt;&gt;<i>     - Example: 10:100891940:N:&lt;NEUTRAL&gt;,10:10
</I>&gt;&gt;<i> 4975905:N:&lt;NEUTRAL&gt;,10:119704960:N:&lt;NEUTRAL&gt;
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of germline.snv.mnv for DO50311 using DKFZ
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 3850992
</I>&gt;&gt;<i> Extra: 0
</I>&gt;&gt;<i> Missing: 0
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of germline.indel for DO50311 using DKFZ
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 709060
</I>&gt;&gt;<i> Extra: 0
</I>&gt;&gt;<i> Missing: 0
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of germline.sv for DO50311 using DKFZ
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 1393
</I>&gt;&gt;<i> Extra: 231
</I>&gt;&gt;<i>     - Example: 10:134319313:N:&lt;DEL&gt;,10:134948
</I>&gt;&gt;<i> 976:N:&lt;DEL&gt;,10:19996638:N:&lt;DEL&gt;
</I>&gt;&gt;<i> Missing: 615
</I>&gt;&gt;<i>     - Example: 10:101851839:N:&lt;TRA&gt;,10:101851
</I>&gt;&gt;<i> 884:N:&lt;TRA&gt;,10:10745225:N:&lt;DUP&gt;
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> File not found /mnt/1TB/work/DockerTest-Migue
</I>&gt;&gt;<i> l/tests/DKFZ/DO50311//output//DO50311.germline.cnv.vcf.gz
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.snv.mnv for DO52140 using DKFZ
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 37160
</I>&gt;&gt;<i> Extra: 0
</I>&gt;&gt;<i> Missing: 0
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.indel for DO52140 using DKFZ
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 19347
</I>&gt;&gt;<i> Extra: 0
</I>&gt;&gt;<i> Missing: 0
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.sv for DO52140 using DKFZ
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 72
</I>&gt;&gt;<i> Extra: 23
</I>&gt;&gt;<i>     - Example: 10:132840774:N:&lt;DEL&gt;,11:382520
</I>&gt;&gt;<i> 19:N:&lt;TRA&gt;,11:47700673:N:&lt;TRA&gt;
</I>&gt;&gt;<i> Missing: 61
</I>&gt;&gt;<i>     - Example: 10:134749140:N:&lt;DEL&gt;,11:179191:N:&lt;TRA&gt;,11:38252005:N:&lt;TRA&gt;
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.cnv for DO52140 using DKFZ
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 275
</I>&gt;&gt;<i> Extra: 94
</I>&gt;&gt;<i>     - Example: 1:106505931:N:&lt;LOH&gt;,1:10906889
</I>&gt;&gt;<i> 9:N:&lt;DEL&gt;,1:109359995:N:&lt;DEL&gt;
</I>&gt;&gt;<i> Missing: 286
</I>&gt;&gt;<i>     - Example: 10:88653561:N:&lt;LOH&gt;,11:179192:N:&lt;LOH&gt;,11:38252006:N:&lt;LOH&gt;
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of germline.snv.mnv for DO52140 using DKFZ
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 3833896
</I>&gt;&gt;<i> Extra: 0
</I>&gt;&gt;<i> Missing: 0
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of germline.indel for DO52140 using DKFZ
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 706572
</I>&gt;&gt;<i> Extra: 0
</I>&gt;&gt;<i> Missing: 0
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of germline.sv for DO52140 using DKFZ
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 1108
</I>&gt;&gt;<i> Extra: 1116
</I>&gt;&gt;<i>     - Example: 10:102158308:N:&lt;DEL&gt;,10:104645
</I>&gt;&gt;<i> 247:N:&lt;DEL&gt;,10:105097522:N:&lt;DEL&gt;
</I>&gt;&gt;<i> Missing: 2908
</I>&gt;&gt;<i>     - Example: 10:100107032:N:&lt;TRA&gt;,10:100107
</I>&gt;&gt;<i> 151:N:&lt;TRA&gt;,10:102158345:N:&lt;DEL&gt;
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> File not found /mnt/1TB/work/DockerTest-Migue
</I>&gt;&gt;<i> l/tests/DKFZ/DO52140//output//DO52140.germline.cnv.vcf.gz
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.snv.mnv for DO50311 using Sanger
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 156299
</I>&gt;&gt;<i> Extra: 1
</I>&gt;&gt;<i>     - Example: Y:58885197:A:G
</I>&gt;&gt;<i> Missing: 14
</I>&gt;&gt;<i>     - Example: 1:102887902:A:T,1:143165228:C:G,16:87047601:A:C
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.indel for DO50311 using Sanger
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 812487
</I>&gt;&gt;<i> Extra: 0
</I>&gt;&gt;<i> Missing: 0
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.sv for DO50311 using Sanger
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 260
</I>&gt;&gt;<i> Extra: 0
</I>&gt;&gt;<i> Missing: 0
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.cnv for DO50311 using Sanger
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 138
</I>&gt;&gt;<i> Extra: 0
</I>&gt;&gt;<i> Missing: 0
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.snv.mnv for DO52140 using Sanger
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 87234
</I>&gt;&gt;<i> Extra: 5
</I>&gt;&gt;<i>     - Example: 1:23719098:A:G,12:43715930:T:A,20:4058335:T:A
</I>&gt;&gt;<i> Missing: 7
</I>&gt;&gt;<i>     - Example: 10:6881937:A:T,1:148579866:A:G,11:9271589:T:A
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.indel for DO52140 using Sanger
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 803986
</I>&gt;&gt;<i> Extra: 0
</I>&gt;&gt;<i> Missing: 0
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.sv for DO52140 using Sanger
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 6
</I>&gt;&gt;<i> Extra: 0
</I>&gt;&gt;<i> Missing: 0
</I>&gt;&gt;<i>
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> Comparison of somatic.cnv for DO52140 using Sanger
</I>&gt;&gt;<i> ---
</I>&gt;&gt;<i> Common: 36
</I>&gt;&gt;<i> Extra: 0
</I>&gt;&gt;<i> Missing: 2
</I>&gt;&gt;<i>     - Example: 10:11767915:T:&lt;CNV&gt;,10:11779907:G:&lt;CNV&gt;
</I>&gt;&gt;<i>
</I>&gt;<i>
</I>&gt;<i>
</I>-------------- next part --------------
An HTML attachment was scrubbed...
URL: &lt;<A HREF="https://lists.icgc.org/mailman/private/docktesters/attachments/20170219/9f75dc01/attachment-0001.html">https://lists.icgc.org/mailman/private/docktesters/attachments/20170219/9f75dc01/attachment-0001.html</A>&gt;
</PRE>


<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="000236.html">[DOCKTESTERS] BWA-Mem validation of HCC1143. 95% matches, 3.6% miss-matches, and 1.3% soft-matches
</A></li>
	<LI>Next message: <A HREF="000238.html">[DOCKTESTERS] Questions about BiasFilter
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#237">[ date ]</a>
              <a href="thread.html#237">[ thread ]</a>
              <a href="subject.html#237">[ subject ]</a>
              <a href="author.html#237">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="https://lists.icgc.org/mailman/listinfo/docktesters">More information about the docktesters
mailing list</a><br>
</body></html>
